# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../firefox/mips64.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- firefox-81.0.2/third_party/rust/authenticator/src/linux/hidwrapper.rs.vanilla	2020-10-18 15:42:50.890221142 +0200
+++ firefox-81.0.2/third_party/rust/authenticator/src/linux/hidwrapper.rs	2020-10-18 15:43:13.658221167 +0200
@@ -17,7 +17,7 @@
 #[cfg(all(target_arch = "mips", target_endian = "big"))]
 include!("ioctl_mipsbe.rs");
 
-#[cfg(all(target_arch = "mips64", target_endian = "little"))]
+#[cfg(target_arch = "mips64")]
 include!("ioctl_mips64le.rs");
 
 #[cfg(all(target_arch = "powerpc", target_endian = "little"))]
--- firefox-81.0.2/third_party/rust/authenticator/build.rs.vanilla	2020-10-18 15:41:03.586221026 +0200
+++ firefox-81.0.2/third_party/rust/authenticator/build.rs	2020-10-18 15:43:25.558221180 +0200
@@ -25,7 +25,7 @@
         "ioctl_mipsbe.rs"
     } else if cfg!(all(target_arch = "mips", target_endian = "little")) {
         "ioctl_mipsle.rs"
-    } else if cfg!(all(target_arch = "mips64", target_endian = "little")) {
+    } else if cfg!(target_arch = "mips64") {
         "ioctl_mips64le.rs"
     } else if cfg!(all(target_arch = "powerpc", target_endian = "little")) {
         "ioctl_powerpcle.rs"
